Changes to cross account role creation process (#1516)
astronomer/docs: astro/cross-account-role-setup.md

| `AttachRolePolicy , DetachRolePolicy`    | `roleName = astronomer-remote-management`                        |
| `SetPolicyVersion , CreatePolicyVersion` | `policyArn = "arn:aws:iam::*:policy/AstronomerCrossAccountRole"` |

> To use a CloudWatch CloudTrail to changes to the cross-account role policy, see [Creating CloudWatch alarms for CloudTrail events](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudwatch-alarms-for-cloudtrail.html).  When you create the metric filter, on the Define pattern page, in Create filter pattern, enter the following for Filter pattern:

Reviewer: ```suggestion
To monitor changes to the cross-account role policy, create an Amazon CloudWatch alarm. See [Creating CloudWatch alarms for CloudTrail events](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudwatch-alarms-for-cloudtrail.html).  

When you create the metric filter, on the Define pattern page, in Create filter pattern, enter the following for Filter pattern:
```
